src/main/java/com/example/dr_app/model/Doctor.java

```diff
@@ -0,0 +1,154 @@
+package com.example.dr_app.model;
+
+import jakarta.persistence.*;
+
+import java.time.LocalDate;
+
+@Entity
+@Table( name = "doctors")
+public class Doctor {
+    @Id
+    @GeneratedValue(
+            strategy = GenerationType.AUTO
+    )
+    @Column(
+            name = "id",
+            updatable = false
+    )
+    private Long id;
+    @Column(
+            name = "first_name",
+            nullable = false,
+            columnDefinition = "TEXT"
+    )
+    private String firstName;
+    @Column(
+            name = "last_name",
+            nullable = false,
+            columnDefinition = "TEXT"
+    )
+    private String lastName;
+    @Column(
+            name = "specialization",
+            nullable = false,
+            columnDefinition = "TEXT"
+    )
+    private String specialization;
+
+    @Column(
+            name = "experience",
+            nullable = false
+    )
+    private Integer experience;
+
+    @Column(
+            name = "contact_number"
+    )
+    private String contactNumber;
+
+    @Column(
+            name = "email",
+            nullable = false,
+            columnDefinition = "TEXT",
+            unique = true
+    )
+    private String email;
+
+    @Column(
+            name = "date_of_birth",
+            nullable = false
+    )
+    private LocalDate dateOfBirth;
+
+    public Doctor() {
+        // Default constructor, why is it needed????
+    }
+
+    public Doctor(String firstName, String lastName, String specialization, Integer experience, String contactNumber, String email, LocalDate dateOfBirth) {
```

Comment:
All this getters setters And constructors can be changed with 
AllArgsConstructor,
Getter,
Setter 

from lombok. 
Please change it.